### PR TITLE
GH-1106 Add Google Gantt Chart

### PR DESCRIPTION
**Pull request by mlzummo**
Adds Google Visualization Library Gantt chart (v1.1)

Fixes GH-1106

Signed-off-by: Dan Snell <Mathew.Zummo@lexisnexis.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
renanme to Materialgantt?

File: src/google/Gantt.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            context.click(context._selection.data, context._selection.column, Object.keys(context._selection).length !== 0);
+        });
+    };
+
+    Gantt.prototype.update = function (domNode, element) {
+        HTMLWidget.prototype.update.apply(this, arguments);
+        this._chart.draw(this.formatData(), this.getChartOptions());
+    };
+
+    function materialHack(callback, depth) {
```
Comment (GordonSmith) [inline src/google/Gantt.js]: Duplicate code - can you common this and the one in the Material bar chart up to a "material" base class.

**Comment by mlzummo** [comment]
@GordonSmith made suggested changes

**Comment by mlzummo** [comment]
@GordonSmith ready to pull i think

**Pull request merged**